crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -779,6 +779,14 @@ impl HolochainP2pActor {
         K2_CONFIG.call_once(|| {
             // Set up some kitsune2 specializations specific to holochain.
 
+            kitsune2_api::OpId::set_loc_callback(|bytes| {
```

Comment:
Uses the new API to implement a different location lookup method to the global lookup method for other id types.